aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/exceptions/AuthExceptionHelper.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.auth.cognito.exceptions
+
+import com.amplifyframework.auth.exceptions.ConfigurationException
+
+internal object AuthExceptionHelper {
+
+    fun createCognitoNotConfiguredException() = ConfigurationException(
+        message = "Auth not configured, cannot process the request.",
```

Comment:
```suggestion
        message = "The Auth plugin is not configured hence we cannot process this request.",
```

Reply:
Merged the other two suggestions but "hence" is not a common word. I don't think our messaging needs to be full sentence structure at all times. I prefer clear, concise messaging over more verbosity/completeness, but this is a discussion we should have to all be in alignment. Next reviewer, please comment opinion.

Reply:
I am fine with the original PR then.

Reply:
I agree that the messages should be short and concise rather than verbose.